### PR TITLE
Improve popup style

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR improves the map popup component with shading and rounded border per https://github.com/opentripplanner/otp-react-redux/pull/667#discussion_r1000960712.

### CODE REVIEW

File: packages/base-map/src/styled.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 import styled from "styled-components";
 
+/**
+ * Adds a box shadow and tweaks border radius to make popups easier to read.
+ */
+export const Popup = styled(MapGlPopup)`
+  & > .maplibregl-popup-content,
+  & > .mapboxgl-popup-content {
+    border-radius: 6px;
+    box-shadow: 0 3px 14px rgb(0 0 0 / 40%);
```
Comment (miles-grant-ibigroup) [inline packages/base-map/src/styled.ts:11]:
```suggestion
    box-shadow: 0 3px 14px 4px rgb(0 0 0 / 40%);
```
Could we add some spread to this?

File: packages/base-map/src/styled.ts [hunk 2]
```diff
@@ -1,5 +1,17 @@
+import { Popup as MapGlPopup } from "react-map-gl";
 import styled from "styled-components";
 
+/**
+ * Adds a box shadow and tweaks border radius to make popups easier to read.
+ */
+export const Popup = styled(MapGlPopup)`
+  & > .maplibregl-popup-content,
+  & > .mapboxgl-popup-content {
+    border-radius: 6px;
```
Comment (miles-grant-ibigroup) [inline packages/base-map/src/styled.ts:10]:
```suggestion
    border-radius: 10px;
```
I think if we bump this up we should commit to a more round look

File: packages/base-map/src/index.tsx [hunk 3]
```diff
@@ -190,4 +190,6 @@ const LayerWrapper = (props: LayerProps): JSX.Element => {
   return <>{visible && children}</>;
 };
 
+export const Popup = Styled.Popup;
```
Comment (miles-grant-ibigroup) [inline packages/base-map/src/index.tsx:193]: I'm not seeing this in the `stop-overlay` or `vehicle-rental-overlay`

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
I like the look but I'd like to see it used in all applicable packages!

File: packages/base-map/src/index.tsx (hunk 3 above)
Comment (binh-dam-ibigroup) [reply packages/base-map/src/index.tsx:193]: These packages need a new version number before they can use the popup with the new style (see `index.tsx` in this PR from the `base-map` package).

Comment (binh-dam-ibigroup) [reply packages/base-map/src/index.tsx:193]: Updated in https://github.com/opentripplanner/otp-ui/pull/471/commits/96fe48e52e433946a26d2b7199136f20574e3443 for storybook viewing only.

File: packages/base-map/src/styled.ts (hunk 2 above)
Comment (binh-dam-ibigroup) [reply packages/base-map/src/styled.ts]: Updated in https://github.com/opentripplanner/otp-ui/pull/471/commits/45ca046b59d4c46d94a43292084bf17a1e50cec0.

File: packages/base-map/src/styled.ts (hunk 1 above)
Comment (binh-dam-ibigroup) [reply packages/base-map/src/styled.ts]: Updated in https://github.com/opentripplanner/otp-ui/pull/471/commits/45ca046b59d4c46d94a43292084bf17a1e50cec0.

Comment (miles-grant-ibigroup) [reply packages/base-map/src/styled.ts]: Sorry I forgot to add one more change I made which is to change 40% to 20%! which I think looks a bit less jarring.

**Review by miles-grant-ibigroup — APPROVED** [review]
With the drop shadow color change this is perfect and the non fix/feat commit on the other packages is the perfect way to get those changes through!

Thanks so much

File: packages/base-map/src/styled.ts (hunk 1 above)
Comment (binh-dam-ibigroup) [reply packages/base-map/src/styled.ts]: In that case could we revert to the initial shadow which has more contrast and a narrower span?

Comment (miles-grant-ibigroup) [reply packages/base-map/src/styled.ts]: I think with 20% instead of 40% the new one looks much more defined! Just needs to be a little more transparent

Comment (binh-dam-ibigroup) [reply packages/base-map/src/styled.ts]: Ok, see https://github.com/opentripplanner/otp-ui/pull/471/commits/539a6ecb056e69a35cbafe6db03d6024676b30ee.

**Review by amy-corson-ibigroup — APPROVED** [review]

**Pull request merged**